amgut/handlers/auth_handlers.py

```diff
@@ -44,8 +57,8 @@ def post(self):
             self.redirect(media_locale['SITEBASE'] + '/db_error/?err=regkit')
```

Comment:
wasn't part of this PR, but this error should probably be put in the text locale, I think